chapter_13_database_layer_validation.asciidoc

```diff
@@ -106,8 +99,6 @@ class ListAndItemModelsTest(TestCase):
 ----
 ====
 
-// DAVID: Catching ValidationError is a bit too much of a catch-all, could
-// we assert what's on the ValidationError for the correct error?
```

Comment:
I think this would be more valid had we expected raising just a generic Exception. In this test case, in my opinion, ValidationError is a sufficiently specific Exception, given `mylist` does have a valid value.
Also, there aren't too many more specific things to assert, in my opinion. We will provide a "You can't have an empty list item" error message but that will be `views`' territory.